packages/store/ts/config/v2/store.test.ts

```diff
@@ -460,9 +468,9 @@ describe("defineStore", () => {
     });
 
     attest<"namespace">(config.namespace).equals("namespace");
-    attest<"namespace">(config.tables.namespace__Example.namespace).equals("namespace");
-    attest<"Example">(config.tables.namespace__Example.name).equals("Example");
-    attest(config.tables.namespace__Example.tableId).equals(
+    attest<"namespace">(config.tables.Example.namespace).equals("namespace");
+    attest<"Example">(config.tables.Example.name).equals("Example");
+    attest(config.tables.Example.tableId).equals(
```

Comment:
We should probably also include the namespaced variant of these for backwards compatibility, but would be considered deprecated usage.